.gitignore

```diff
@@ -8,4 +8,5 @@ _algolia_api_key
 _pdf
 _site/
 node_modules/
-vendor
\ No newline at end of file
+vendor
+.cb
```

Comment:
Minor, just curios—what’s created .cb folders?

Reply:
This is an extension I use in vscode I use that to add a banner per project in vscode

Reply:
Yes it works, tested responsive

Reply:
Cool! Thanks for clarifying and confirming! :)